### PR TITLE
Suggestions from Rosie

### PR DESCRIPTION
**Pull request by rosiel**
Submitted in partial fulfillment of More Eyeballs (#3) 

I gave this a "what would I like to see here" review, which may be over-explaining things that would be better left for the reader to figure out, and may be not explaining things enough.

Also, 
- table of contents
- a few typos

Feel free to counter-suggest, e.g. if i'm wrong or over-verbose.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,7 +210,7 @@ destination:
   plugin: 'entity:file'
   type: image
 ```
-You can create any type of content entity in Drupal. In this case, we're making file entities.  Specifically, we're making images, which are a special type of file entity.
+You can create any type of content entity in Drupal. In this case, we're making file entities.  Specifically, we're making images, which are a special type of file entity that's provided by core Drupal.
```
Comment (rosiel) [inline README.md]: Image is part of core, right?

File: README.md [hunk 2]
```diff
@@ -568,6 +627,6 @@ If you've made it all the way to the end here, then you've learned that you can
...
+But really the best thing to do is try and get your data into Islandora!  We intend to create a `boilerplate` branch of this repository that will allow you to clone down a migration template, ready for you to customize to fit your data.  And as you assemble it into CSV format, keep in mind that if you have more than just names for things like subjects and authors, that you can always make more CSVs.  Think of it like maintaining tables in an SQL database.  Each CSV has unique keys, so you can lookup/join entiities between CSVs using those keys.  And you can still pipe-delimit the keys like we did in our example to handle multi-valued fields.
```
Comment (rosiel) [inline README.md]: yeah I don't see a `boilerplate` branch yet! I figure we shouldn't lie until we have one shipped.

File: README.md (hunk 1 above)
Comment (dannylamb) [reply README.md]: :+1: yes

File: README.md (hunk 2 above)
Comment (dannylamb) [reply README.md]: done!

**Pull request merged**